models/LM-tensorflow/run.py

```diff
@@ -76,7 +76,16 @@ def run():
 	random.seed(0)
 
 	from main import main
+
 	main(args)
+	if args.mode == 'test':
+		import os
+		import json
+		res = {'working_dir': './', 'entry': 'run', 'args': argv}
```

Comment:
It is a little confusing. In pratical, the result.json is generated by our lib, not by model.
Here, we should just place a result.json rather than generate one.